README.md

```diff
@@ -152,7 +152,7 @@ babili     21.9kB  219%    7.46kB 164%     2ms        722ms
 closure js 24.01kB 191%    8.04kB 144%     2ms        3301ms
 ```
 
-Run with: `./scripts/benchmark.js ./scripts/fixtures/backbone.js`
+Run with: `./scripts/benchmark.js backbone` **or** `./scripts/benchmark.js backbone backbone/backbone.js`
```

Comment:
I think you're usage example already covers the syntax pretty well so I don't think the alternative form is required. I'm assuming these lines are just to demonstrate how the results were generated. Therefore we should try and work out which npm package version corresponds to the fixtures files that were previously in use and specify those e.g. `./scripts/benchmark.js backbone@x.y.z`.